flink-table/flink-sql-parser/pom.xml

```diff
@@ -156,6 +174,12 @@ under the License.
 			<scope>test</scope>
 			<type>test-jar</type>
 		</dependency>
+		<dependency>
+			<groupId>org.junit.jupiter</groupId>
+			<artifactId>junit-jupiter-engine</artifactId>
```

Comment:
Because Calcite tests use Junit5 and we extend from it's `SqlParserTest`, add this dependency to make Junit4 tests compatible with Junit5 environment.